test/test_003_path_like_division.py

```diff
@@ -2,35 +2,35 @@
 
 
 def test_issue_003_path_like_division_trailing():
-	base = URI("http://example.com/foo/")
-	assert str(base) == "http://example.com/foo/"
-	assert str(base / "bar.html") == "http://example.com/foo/bar.html"
-	
-	base = URI("http://example.com/foo")
-	assert str(base) == "http://example.com/foo"
-	assert str(base / "bar.html") == "http://example.com/bar.html"
+    base = URI("http://example.com/foo/")
+    assert str(base) == "http://example.com/foo/"
+    assert str(base / "bar.html") == "http://example.com/foo/bar.html"
+
+    base = URI("http://example.com/foo")
+    assert str(base) == "http://example.com/foo"
+    assert str(base / "bar.html") == "http://example.com/bar.html"
```

Comment:
All of the tabs-to-spaces nonsense makes it far more difficult to identify what has actually changed. I use tabs. All Marrow projects use tabs. Why use four bytes when one will do? Prior to release of this update I'll have to go through it and correct this, which will nuke some attribution on lines that actually have intentionally changed.

Similar comment coming up in the next chunk…